lib/util/search.js

```diff
@@ -307,6 +303,7 @@ SearchClient.prototype.prepareBodyForCreate = function(body, page, index = null)
     path: page.path,
     body: page.revision.body,
     username: page.creator.username,
+    grant: page.grant,
```

Comment:
@lightnet328 It seems the schema of doc on es has been changed.
So should we apply this change to `mappings.json` ?

Reply:
I think so too. I'll add grant field to `mapping.json`.